addons/external_intercom/ui/tfar_ace_interaction_external_intercom_phone.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These don't need to be inside a pbo, waste a bit of space
We have other source files in https://github.com/michail-nikolaev/task-force-arma-3-radio/tree/master/extra
but this is small enough to not matter